config.toml

```diff
@@ -2,11 +2,11 @@
 owner = "brejoc"
 name = "test"
 updateInterval = 3600
+boardList = ["test", "test2"]
```

Comment:
How about structuring the config like this? https://github.com/brejoc/filtra/pull/19/files#diff-618063036395fe9ee107f22b46c9eadeR6-R19

Reply:
It certainly works. I can adjust the PR if you see major advantages.

I have used a simpler approach to make configuration easier, and because labels are common, as well as most column names.

Reply:
I don’t think it’s save to assume that this stays like this forever. Also if someone else wants to use filtra we can’t make this assumption for them.

Reply:
I can understand that argument for board-specific settings, like column names. However, in general, is it desirable that we identify bugs with different labels within the same repo? Same logic applies to supportLabels. I feel it would be confusing to have different ways of identifying those across boards that belong to the same repo.

Reply:
My suggestions would therefore be something to the effect of: 
```
[repository]
owner = "brejoc"
name = "test"
updateInterval = 3600
bugLabels       = ["bug"]
supportLabels   = ["L3", "L3 question"]

[boards]

  [test]
  plannedColumns  = ["Requested", "Planned"]
  blockedColumns  = ["Blocked / Postponed", "Waiting for Request"]

  [test2]
  plannedColumns  = ["Todo"]
  blockedColumns  = ["Blocked"]
```

Reply:
Unit tests passing. Ready for final review,